root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4208,6 +4208,11 @@ const testData = [
             expected_clean_url: 'https://www.youtube.com/playlist?list=PL43OynbWaTMKSxLVnUF0HbHHiXEgAVm3Q',
        only_valid_entity_types: ['artist', 'event', 'label', 'place', 'series'],
   },
+  {
+                     input_url: 'https://music.youtube.com/browse/MPREb_0bOFkwXrX2x',
+             input_entity_type: 'release',
+    expected_relationship_type: undefined,
+  },
```

Comment:
Test should be extended as follows to make sure that URL is not incidentally changed by some rule and that manually selecting `youtube` relationship type is disallowed too.

```suggestion
  {
                     input_url: 'https://music.youtube.com/browse/MPREb_0bOFkwXrX2x',
             input_entity_type: 'release',
    expected_relationship_type: undefined,
            expected_clean_url: 'https://music.youtube.com/browse/MPREb_0bOFkwXrX2x',
       input_relationship_type: 'youtube',
       only_valid_entity_types: [],
  },
```